docs/nitclk/class.rst

```diff
@@ -461,17 +461,17 @@ exported_sync_pulse_output_terminal
 
         The following table lists the characteristics of this property.
 
-            +----------------+------------+
-            | Characteristic | Value      |
-            +================+============+
-            | Datatype       | str        |
-            +----------------+------------+
-            | Permissions    | read-write |
-            +----------------+------------+
-            | Channel Based  | No         |
-            +----------------+------------+
-            | Resettable     | No         |
-            +----------------+------------+
+            +-----------------------+------------+
+            | Characteristic        | Value      |
+            +=======================+============+
+            | Datatype              | str        |
+            +-----------------------+------------+
+            | Permissions           | read-write |
+            +-----------------------+------------+
+            | Repeated Capabilities | None       |
```

Comment:
There is no concept of Repeated Capabilities in `nitclk`.
Or `nimodinst` for that matter.
NI-DMM has the concept but it isn't applicable to the `nidmm` Python module.

I'm thinking we might be better off not even displaying this row at all for APIs without repeated capabilities.

I think Resettable, while a valid thing, should not be here either because the Python APIs have not exposed it (most driver runtimes don't implement it I believe)

#1606